dummy_robot/dummy_map_server/README.md

```diff
@@ -0,0 +1,96 @@
+## **What Is This?**
+
+This demo creates and runs a ROS2 **node**, `dummy_map_server`, that publishes `nav_msgs::msg::OccupancyGrid` on a ROS2 publisher topic, `/map`.
+
+**dummy_map_server** is also a dependency of **dummy_robot_bringup**. Please refer to [dummy_robot_bringup](https://github.com/ros2/demos/tree/rolling/dummy_robot/dummy_robot_bringup).
+
+## **What Is An Occupancy Grid?**
+
+An occupancy grid is essentially a **2-dimensional** map with each cell containing an integer to indicate the presence, absence or general state of a space in a certain physical room a robot is in. With this set of information, it better informs how a robot can best navigate through obstacles.
+
+Eg.
```

Comment:
I wonder if this can be worded more clearly. Here's my try. Feel free to iterate further.

```suggestion
An occupancy grid can be thought of as a map of an environment that has been divided into grid cells, where each grid cell is either occupied or unoccupied (in advanced applications, they can have other states, too).
This map of occupied and unoccupied grid cells can be used to help a robot navigate a space, for example to drive through a room with furniture.
 If a robot is navigating a new room, however, there may also be grid cells that are marked as unknown, until that area is explored.
 These different states are often stored in the occupancy grid as integers, for example:
```

A few points of feedback:
- "**2-dimensional*": Can't they be 3-d or 1-d, as well? 
- "a certain physical room a robot is in": it's not necessarily true that there needs to be a robot. It could also be a handheld sensor that generates an occupancy map of the space.
- I prefer to write things out. We have some users who are not native English speakers and I imagine acronyms add confusion, so I'd rather just write, "For example."